s3tests_boto3/functional/test_s3.py

```diff
@@ -13301,3 +13301,207 @@ def test_delete_bucket_encryption():
 
     response = client.delete_bucket_encryption(Bucket=bucket_name)
     eq(response['ResponseMetadata']['HTTPStatusCode'], 204)
+
+@attr(assertion='success')
+@attr('encryption')
+def _test_sse_s3_customer_write(file_size):
+    """
+    Test enables bucket encryption.
+    Create a file of A's of certain size, and use it to set_contents_from_file.
+    Re-read the contents, and confirm we get A's
```

Comment:
confirm we get A's -> confirm we get same A's as input or some other wording to indicate output as input.

Reply:
Addressed in the commit